### ISSUE TITLE
Does this model support the segmentation file?

### ISSUE THREAD

**Issue by YuqingWang1029** [fundamentalvision/Deformable-DETR#11]

**Comment by jackroos** [comment]
Hi @Epiphqny 

It is from the original DETR code and we have not tested it or train the segmentation model. But I think it is not a big problem to train the segmentation model following the instructions in [DETR repo](https://github.com/facebookresearch/detr).

**Comment by YuqingWang1029** [comment]
@jackroos Thanks for your quick reply, I will have a try.

**Comment by YuqingWang1029** [comment]
@jackroos How should I adjust the learning rate if I want to use larger batch size? Do you have any advice?

**Comment by jackroos** [comment]
@Epiphqny In my experience, when adam is used, you should keep the learning rate unchanged if the batch size does not change a lot. But for a much larger batch size, you may need to tune the learning rate by yourself.
